test_views.py

```diff
@@ -73,74 +73,42 @@ def test_invalid_weather_url_invalid_address(self, is_place_query_valid,):
         self.assertTrue(is_place_query_valid.called)
         self.assertEqual(result.status_code, 400)
 
-    @patch('app.geocode_api_parser.Geocode.check_geocode_api_is_present')
-    @patch('app.weather_parser.WeatherForecast.check_forecast_api_is_present')
+    @patch('app.geocode_api_parser.Geocode.get_address_query')
     @patch('app.weather_parser.WeatherForecast.generate_forecast')
-    @patch('app.geocode_api_parser.Geocode.get_coordinates')
-    @patch('app.geocode_api_parser.Geocode.get_place_info')
-    @patch('app.geocode_api_parser.Geocode.is_place_query_valid')
-    def test_valid_weather_url_valid_address(self, is_place_query_valid, get_place_info, get_coordinates, generate_forecast, check_forecast_api_is_present, check_geocode_api_is_present):
-        check_geocode_api_is_present.return_value = True
-        check_forecast_api_is_present.return_value = True
-        is_place_query_valid.return_value = True
-        get_coordinates.return_value = {'lat': 10.3338299, 'lng': 123.8941434}
-        get_place_info.return_value = {
-            'formatted_address': 'Lahug, Cebu City, Cebu, Philippines',
-            'place_type': 'neighborhood'
-        }
+    def test_valid_weather_url_valid_address(self, generate_forecast, get_address_query):
+        get_address_query.return_value = get_address_query.return_value = {'results': [{'geometry': {'location': {'lat': 10.3338299, 'lng': 123.8941434}, 'bounds': {'southwest': {'lat': 10.3190787, 'lng': 123.878746}, 'northeast': {'lat': 10.3492015, 'lng': 123.9091301}}, 'location_type': 'APPROXIMATE', 'viewport': {'southwest': {'lat': 10.3190787, 'lng': 123.878746}, 'northeast': {'lat': 10.3492015, 'lng': 123.9091301}}}, 'place_id': 'ChIJA7wyKi-ZqTMRPmyxevF67Yw', 'types': ['neighborhood', 'political'], 'formatted_address': 'Lahug, Cebu City, Cebu, Philippines', 'address_components': [{'short_name': 'Lahug', 'types': ['neighborhood', 'political'], 'long_name': 'Lahug'}]}], 'status': 'OK'}
```

Comment:
I'm referring to `get_address_query.return_value = get_address_query.return_value = ...`